src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala

```diff
@@ -43,6 +43,8 @@ object ScapegoatSbtPlugin extends AutoPlugin {
       Defaults.compileSettings ++
       Seq(
         sources := (sources in Compile).value,
```

Comment:
@JoshRosen The short answer is that it worked :smile: . But I think what happened is I made the same assumption you made (that inheriting Compile should have propagated the settings), but in practice that didn't seem to happen. I don't know that I searched terribly hard for a cleaner solution, though.